apps/accounts/views.py

```diff
@@ -18,3 +35,54 @@ def disable_user(request):
     user.save()
     logout(request)
     return Response(status=status.HTTP_200_OK)
+
+
+class PasswordResetTokenView(GenericAPIView):
+    """
+    Password reset e-mail link is confirmed, therefore
+    this resets the user's password.
+
+    Accepts the following POST parameters: token, uid,
+        new_password1, new_password2
+    Returns the success/fail message.
+    """
+    serializer_class = PasswordResetTokenSerializer
+
+    permission_classes = (AllowAny,)
+
+    def post(self, request):
+        self.get_serializer(data=request.data).is_valid(raise_exception=True)
+        return Response({'data': request.data})
+
+
+class PasswordResetConfirmView(GenericAPIView):
+    """
+    Password reset e-mail link is confirmed, therefore
+    this resets the user's password.
+
+    Accepts the following POST parameters: token, uid,
+        new_password1, new_password2
+    Returns the success/fail message.
+    """
+    serializer_class = PasswordResetConfirmSerializer
+    permission_classes = (AllowAny,)
+
+    @sensitive_post_parameters_m
+    def dispatch(self, *args, **kwargs):
+        return super(PasswordResetConfirmView, self).dispatch(*args, **kwargs)
+
+    def post(self, request):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        if serializer.is_valid():
+            uid = force_text(uid_decoder(request.data['uid']))
+            host = request.data['host']
+            send_mail('Password reset on '+host,
```

Comment:
Can we please define the email body in a constants file like `constants.py`. The code would look clean then.